src/Main.purs

```diff
@@ -177,21 +177,21 @@ commands = [
       Args.option "standalone" ["--standalone"] Type.string
         "Output a UMD bundle with the given external module name.",
       Args.option "skipCompile" ["--skip-compile"] Type.flag
-        "Assume PureScript code has already been compiled. Useful for when you want to pass options to psc."
+        "Assume PureScript code has already been compiled. Useful for when you want to pass options to purs."
       ] <> buildArgs,
   Args.command "run" "Compile and run the project." remainderToProgram Run.action $ [
     Args.optionDefault "runtime" ["--runtime", "-r"] Type.string
       "Run the program using this command instead of Node." "node"
     ] <> runArgs,
-  Args.command "docs" "Generate project documentation." remainderToPscDocs Docs.action $ [
+  Args.command "docs" "Generate project documentation." remainderToDocs Docs.action $ [
     Args.option "withTests" ["--with-tests", "-t"] Type.flag
       "Include tests.",
     Args.option "withDependencies" ["--with-dependencies", "-d"] Type.flag
       "Include external dependencies."
     ] <> pathArgs,
-  Args.command "psci"
-    "Launch a PureScript REPL configured for the project." remainderToPsci
-    Psci.action pathArgs,
+  Args.command "repl"
```

Comment:
I don't think it would be too bad to have the alias show up in the help, but on the other hand it would be nice to come up with a more general solution for command aliases.